app/views/products/_form.html.erb

```diff
@@ -0,0 +1,37 @@
+ 
+   <div class="container">
+   <%= simple_form_for product do |f| %>
+  <div class="row">
+    <div class="col">
+      <%= f.input :title %>
+    </div>
+    #need to add description 
+  </div>
+  <div class="form-row">
+    <div class="form-group col-md-3">
+      <%= f.input :status %>
```

Comment:
Hi! I think we don't need the user to input the status of the product as the status will be 'Pending' by default

Reply:
Hi, yes great spot thank you! Made a trial form to test the view. We can add and remove fields and design the type of form today according to how we want it to look. Will make the corrections today